src/components/Navigation/NavigationList/_navigationList.scss

```diff
@@ -1,7 +1,7 @@
 @use '#scss/allPartials' as *;
 
-$opacity-transition: opacity 0.25s ease-in;
-$top-transition: top 0.25s ease-in 0.1s;
+$opacity-transition: opacity 0.2s cubic-bezier(0.65, 0, 0.35, 1);
+$top-transition: top 0.3s cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
Are we using these anywhere?

Reply:
If you expand the preview, they are used directly below in this file. They were added as a part of another PR so those changes don't show in my PR